budget/__main__.py

```diff
@@ -196,6 +228,21 @@ def main():
             transaction[0] = str(transaction[0])[:10]
             insertTransaction(transaction, service, command, monthlySheetId, summary.title)
             return
+        if command == "edit":
+            subcommand = param[0]
+            lineIndex = int(param[1])
+            transaction, dateAdded = parseTransaction(param[2])
+            monthlySheetId = getMonthlySheetId(transaction[0], sheetIds)
+            transactions = readTransactions(service, monthlySheetId, subcommand)
+            validateLineIndex(lineIndex, transactions)
+            if dateAdded is True:
+                adjustEditedTransactionDate(transaction, lineIndex, transactions)
```

Comment:
Let's just replace this with:
```py
if noExplicitDate is True:
    transaction[0] = transactions[lineIndex - 1][0]
```

and remove `adjustEditedTransactionDate` altogether, just for simplicity.

(I think it would be better if we renamed `dateAdded` as `noExplicitDate` for readability.)